core/domain/html_validation_service.py

```diff
@@ -938,6 +938,83 @@ def validate_math_tags_in_html(html_string):
     return error_list
 
 
+def validate_math_tags_in_html_for_new_schema(html_string):
```

Comment:
No one is going to know what "new schema" means -- relative terms like "old" and "new" only make sense within this PR. Use a more explicit name that stands the test of time.

Reply:
done